.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,17 @@
+---
+name: default
+about: Default Issue Template
+title: ""
+labels: ""
+assignees: ""
+---
+
+## Description
+
+Explain short what you are going to do in this issue.

```

Comment:
Perhaps consider revising for clarity to: 
Please briefly explain what you intend to do about this issue.